--- /dev/null	2015-07-20 21:41:08.858000000 +0000
+++ Python-3.3.6/Lib/test/dh1024.pem	2015-09-01 22:53:24.275000000 +0000
@@ -0,0 +1,7 @@
+-----BEGIN DH PARAMETERS-----
+MIGHAoGBAIbzw1s9CT8SV5yv6L7esdAdZYZjPi3qWFs61CYTFFQnf2s/d09NYaJt
+rrvJhIzWavqnue71qXCf83/J3nz3FEwUU/L0mGyheVbsSHiI64wUo3u50wK5Igo0
+RNs/LD0irs7m0icZ//hijafTU+JOBiuA8zMI+oZfU7BGuc9XrUprAgEC
+-----END DH PARAMETERS-----
+
+Generated with: openssl dhparam -out dh1024.pem  1024
--- Python-3.3.6/Lib/test/test_ssl.py	2014-10-12 07:03:53.000000000 +0000
+++ Python-3.3.6/Lib/test/test_ssl.py.patch	2015-09-01 22:54:34.143000000 +0000
@@ -57,7 +57,7 @@
 NOKIACERT = data_file("nokia.pem")
 NULLBYTECERT = data_file("nullbytecert.pem")
 
-DHFILE = data_file("dh512.pem")
+DHFILE = data_file("dh1024.pem")
 BYTES_DHFILE = os.fsencode(DHFILE)
 
 def handle_error(prefix):
